cards-examples/Web/Web-card-article.md

```diff
@@ -10,229 +10,249 @@ path: /
+      <div class="mdc-button__ripple"></div>
+      <span class="mdc-button__label">Read</span>
+    </button>
+    <button class="mdc-button mdc-card__action mdc-card__action--button">
+      <div class="mdc-button__ripple"></div>
+      <span class="mdc-button__label">Bookmark</span>
+    </button>
+  </div>
+  <div class="mdc-card__action-icons">
+   <button class="material-icons mdc-icon-button mdc-card__action mdc-card__action--icon" title="Share">share</button>
+    <button class="material-icons mdc-icon-button mdc-card__action mdc-card__action--icon" title="More options">more_vert</button>
+  </div>
+</div>
+```
 
-API and source code
+##### Icons
 
-* `\<card classes\>`
-  * [Class definition](https://)
-  * [GitHub source](https://github.com/material-components/)
+We recommend using [Material Icons](https://material.io/tools/icons/) from Google Fonts:
 
-_Use the [Shrine theme](https://material.io/design/material-studies/shrine.html) for this example_
+```html
+<head>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
 ```
-* Display an outlined card 
-* Display a card title "Card title"
-* Display a secondary title "Secondary text"
-* Display text reading "Greyhound divisively hello coldly wonderfully marginally far upon excluding."
-* Display two actions, "Action 1" and "Action 2" with two text buttons
-* Display the sample image ![sample card image of yellow and red tulips](assets/card-sample-image.jpg)
-* Make the card checkable with a "favorites" icon
 
+However, you can also use SVG, [Font Awesome](https://fontawesome.com/), or any other icon library you wish.
+
+#### Combined Example
+
+The following is an example incorporating all of the above elements:
+
+```html
+<div class="mdc-card">
+  <div class="mdc-card__primary-action">
+    <div class="mdc-card__media mdc-card__media--square">
+      <div class="mdc-card__media-content">Title</div>
+    </div>
+    <!-- ... additional primary action content ... -->
+  </div>
+  <div class="mdc-card__actions">
+    <div class="mdc-card__action-buttons">
+      <button class="mdc-button mdc-card__action mdc-card__action--button">
+        <div class="mdc-button__ripple"></div>
+        <span class="mdc-button__label">Action 1</span>
+      </button>
+      <button class="mdc-button mdc-card__action mdc-card__action--button">
+        <div class="mdc-button__ripple"></div>
+        <span class="mdc-button__label">Action 2</span>
+      </button>
+    </div>
+    <div class="mdc-card__action-icons">
+      <button class="material-icons mdc-icon-button mdc-card__action mdc-card__action--icon" title="Share">share</button>
+      <button class="material-icons mdc-icon-button mdc-card__action mdc-card__action--icon" title="More options">more_vert</button>
+    </div>
+  </div>
+</div>
 ```
-* Make the card checkable with a "favorites" icon
-
 
-## `mdc-card` API
+## API
 
 ## Style Customization
```

Comment:
Use sentence case for titles: capitalize the first word, and only capitalize following proper nouns.